wiki/content/howto/index.md

```diff
@@ -264,6 +264,50 @@ The debug output also shows UserMeta information:
 - `{item}`: Item posting list
 - `{deleted}`: Delete marker
 
+## Using the Dgraph Sentry Integration
+
+Sentry is a powerful service that allows applications to send arbitrary events, messages, exceptions, bread-crumbs (logs) to your sentry account. In simplest terms, it is a dial-home service but also has a  rich feature set including event filtering, data scrubbing, several SDKs, tagging, environment and release tagging, and integration with Slack, GitHub etc.
+
+Sentry reporting is on by default. However, starting from v20.03.1 and v20.07.0, there is a configuration flag “enable-sentry” which can be used to completely turn off Sentry events reporting. 
+
+### Basic Integration
+
+**Panics (runtime and manual)**
+
+* For manual panics anywhere in the code, sentry.CaptureException() API is called. 
+
+* For runtime panics, Sentry does not have any native method. After further research we chose the approach of a wrapper process to capture these panics. The basic idea for this is that whenever a dgraph instance is started, a 2nd monitoring process is started whose only job is to monitor the stderr for panics of the monitored process. When a panic is seen, it is reported back to sentry via the CaptureException API. 
+
+**Reporting**
+
+Each event is tagged with the release version, environment, timestamp, tags as explained below. 
+
+**Release:**
+
+  - This is the release version string of the Dgraph instance.
+  
+**Environments:**
+
+We have defined 4 environments:
+
+**dev-oss / dev-enterprise**: These are events seen on non-released / local developer builds.
+
+**prod-oss/prod-enterprise**: These are events on released version such as v20.03.0. Events in this category are also sent on the sentry-events slack channel.
+
+**Tags:**
+
```

Comment:
Tags are key-value pairs  that provide additional context for an event. Tags are sent along with the event.